learners/nonuniform_quantization/learner.py

```diff
@@ -255,10 +255,17 @@ def __build_train(self):
           if v not in clusters]
 
       # determine the var_list optimize
-      if FLAGS.nuql_opt_mode == 'both':
-        optimizable_vars = self.trainable_vars
-      elif FLAGS.nuql_opt_mode == 'clusters':
-        optimizable_vars = clusters
+      if FLAGS.nuql_opt_mode in ['cluster', 'both']:
+        if FLAGS.nuql_opt_mode == 'both':
+          optimizable_vars = self.trainable_vars
+        else:
+          optimizable_vars = clusters
+        if  FLAGS.nuql_enbl_rl_agent:  
```

Comment:
There are **two** whitespaces between `if` and `FLAGS.nuql_enbl_rl_agent`. Please reduce to **one**.